adhocracy4/comments/static/comments/Comment.jsx

```diff
@@ -8,12 +8,15 @@ var CommentManageDropdown = require('./CommentManageDropdown')
 
 var React = require('react')
 var django = require('django')
-var moment = require('moment')
 
 var safeHtml = function (text) {
   return { __html: text }
 }
 
+var localeDate = function (dateStr) {
+  return new Date(dateStr).toLocaleString(document.documentElement.lang)
```

Comment:
Basically there are three options here:

  1. Enforce the ECMA script date format on the DateTimeFields of the API
  2. Require all projects to set their `REST_FRAMEWORK = { ' DATETIME_FORMAT': '???' }`
  3. Continue using the current iso format and replace ` ` with `T` before parsing.